apps/storybook/stories/Namekit/CurrencySymbol.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import { Currency } from "@namehash/ens-utils";
+import { PriceDisplaySize, CurrencySymbol } from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+export const ETH: Story = {
+  args: {
+    currency: Currency.Eth,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const USD: Story = {
+  args: {
+    currency: Currency.Usd,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const USDC: Story = {
+  args: {
+    currency: Currency.Usdc,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const WETH: Story = {
+  args: {
+    currency: Currency.Weth,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const DAI: Story = {
+  args: {
+    currency: Currency.Dai,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const MicroSize: Story = {
```

Comment:
MicroSize and SmallSize appear exactly the same to me on my laptop. This seems confusing.

Reply:
Indeed these have the same width and height and this spec came from artboards: 

Uploading Gravação de Tela 2024-07-10 às 21.25.56.mov…

Reply:
https://github.com/namehash/nameguard/assets/49823133/c1cb4fca-15c3-4c7f-a703-1f31655a84d6

Reply:
@FrancoAguzzi I see that the icon size is the same, but the font size / styling is different.

How do you suggest we manage this?

In my mind, the icon should only have 2 size enum values if it only actually has 2 displayed sizes.

Reply:
That's correct !

I suggest we map every PriceDisplaySize to an icon display size, even if these are the same across different display size options. 

The idea behind having only 2 enum options would unfortunately not fit the complete idea of this component since PriceDisplaySize have 4 diff options. Supporting only 2 **requires** changing the type of this prop for full support. We'd then change the mapping of the chosen DisplayedPrice size from PriceDisplaySize --> PriceDisplaySize to PriceDisplaySize --> string. The abstraction of today reuses the same PriceDisplaySize type in the communication between DisplayedPrice and CurrencySymbol. The new option would do the other mapping mentioned 

That's correct. once again!

(respectively answering ;))

Reply:
@FrancoAguzzi This is still an open question on this PR.